### PR TITLE
Absolute post ordering

### PR DESCRIPTION
**Pull request by UnwarySage**
Please answer the following questions and leave the below in as part of your PR.

- [x] This PR corresponds to an [issue with a clear problem statement](https://github.com/babashka/babashka/blob/master/doc/dev.md#start-with-an-issue-before-writing-code).

#66 for reference

- [x] I have updated the [CHANGELOG.md](https://github.com/babashka/babashka/blob/master/CHANGELOG.md) file with a description of the addressed issue.

I'd have added tests, but the relevant areas of test suite seem to be having issues and are commented out.

### CODE REVIEW

File: src/quickblog/internal.clj [hunk 1]
```diff
@@ -142,8 +142,26 @@
         :html
         post-process-markdown)))
 
+(defn post-compare [a-post b-post]
```
Comment (borkdude) [inline src/quickblog/internal.clj:145]: There's probably an easier way of writing this?

E.g. compare the vector [date title file]?

```
user=> (compare ["b" "b" "a"] ["b" "b" "a"])
0
user=> (compare ["b" "b" "a"] ["b" "b" "b"])
-1
user=> (compare ["b" "b" "a"] ["b" "b" "a"])
1
```

**Comment by UnwarySage** [comment]
Hmm, would have to make the dates negative in the vector to stop everything
from being sorted descending. Will give it a try.

**Comment by UnwarySage** [comment]
Had a post push epiphany. Here's a variant like you suggested, but using opposite positions to have dates sorted in descending order.

**Comment by borkdude** [comment]
Thanks!

**Pull request merged**